cmd/neofs-adm/internal/modules/morph/generate.go

```diff
@@ -35,6 +35,9 @@ func generateAlphabetCreds(cmd *cobra.Command, _ []string) error {
 	if err != nil {
 		return err
 	}
+	if cmd.Name() != "init" {
```

Comment:
I don't think this is needed and it's likely to break `generate-alphabet` command.